src/components/Monitoring/index.tsx

```diff
@@ -152,7 +152,7 @@ export default function Monitoring(props: MonitoringProps): JSX.Element {
       <Grid container>
         {hasSeedBanks ? (
           <>
-            <PageHeaderWrapper nextElement={contentRef.current}>
+            <PageHeaderWrapper nextElement={contentRef.current} nextElementInitialMargin={-24}>
```

Comment:
may want to skip the negative margin on mobile, it seems to be blending into the seedbank picker

Reply:
Left it in, and added the margin to the header for mobile. It looks correct now.